.circleci/config.yml

```diff
@@ -641,12 +638,32 @@ jobs:
                 -DPIKA_WITH_TESTS_DEBUG_LOG=On \
                 -DPIKA_WITH_TESTS_DEBUG_LOG_DESTINATION=/build/debug-log-arm.txt \
                 -DPIKA_WITH_SPINLOCK_DEADLOCK_DETECTION=On
-          working_directory: /home/circleci/project/build
+          working_directory: $HOME/project/build
       - run:
-          name: Build pika Arm64
+          name: Building pika Arm64
           command: |
             ninja -k 0 pika
-          working_directory: /home/circleci/project/build
+          working_directory: $HOME/project/build
+      - run:
+          name: Building tests Arm64
+          command: |
+            ninja -k 0 tests.unit.modules.assertion
+          working_directory: $HOME/project/build
+      - run:
+          name: Running tests Arm64
+          when: always
+          command: |
+              ulimit -c unlimited
+              ctest -R tests.unit.modules.assertion \
```

Comment:
This is just a reminder comment to enable running all tests before merging (same for building).